src/StatisticsClaimed.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace DDB\Stats;
+
+use Carbon\Carbon;
+
+class StatisticsClaimed
+{
+    protected $dateTime;
+
+    public function __construct(\DateTimeInterface $dateTime = null)
+    {
+        $this->dateTime = $dateTime ?? Carbon::now();
```

Comment:
Why make dateTime optional? Defaulting to now makes sense in the controller, but here?

Reply:
:+1: I removed the default and tried to add some clarification of the meaning of the parameter in a DocBlock.